resources/xsd/fastRTPS_profiles.xsd

```diff
@@ -172,7 +172,7 @@
     <xs:complexType name="publisherProfileType">
         <xs:all>
             <xs:element name="topic" type="topicElementType" minOccurs="0" maxOccurs="1"/>
-            <xs:element name="qos" type="QosPoliciesType" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="qos" type="publisherQosPoliciesType" minOccurs="0" maxOccurs="1"/>
```

Comment:
I suggest using DDS terminology: `dataWriterQosPoliciesType` and `dataReaderQoSPoliciesType`. Does it make sense to have the common Qos Policies in the previous type and define only a new QoS type with the specific ones that apply only to DataWriters? Can this be done?

Reply:
Yes, it makes sense